.github/workflows/cypress-ete.yml

```diff
@@ -0,0 +1,35 @@
+name: cypress-ete
+on: push
+jobs:
+  cypress-ete:
+    name: Cypress (ete)
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Cypress run
+        uses: cypress-io/github-action@v2
+        env:
+          PORT: 8861
+          NODE_ENV: development
+          ENVIRONMENT: production
+          STAGE: production
```

Comment:
stage in this case would be what the guardian use

so either `DEV`, `CODE`, or `PROD`

think in this case you want `CODE`

Reply:
You have hawk eyesight!

Reply:
Yeah that's right, `CODE` is pretty much what we call our staging/manual testing environments. In the `yml` file you can see what CODE urls look like, generally in the form `*.code.dev-theguardian.com`

Reply:
`CODE` is the name for the test/stage/pre prod/qa environment. It is all of those rolled into one. Not sure why it's called 'CODE' - I think there's a story there

See: https://m.code.dev-theguardian.com/uk

Reply:
> I'm going to be v annoying and ask loads of questions while I'm still new

ALWAYS ask questions - never annoying

Reply:
lovely, thanks guys